### PR TITLE
skunert - Retain Rank II Jan - May 2024

### PR DESCRIPTION
**Pull request by skunert**
Hey everyone,

I am submitting evidence here to retain my current rank II.

Hash: `abd60508482a3f5ecf69f8e23b3d18c04a182b083ce7d87d13ab02d9d7c2ab63`

How to verify using [blake2](https://crates.io/crates/blake2_bin):
```
blake2 -s --length 32 evidence/skunert/0001-skunert-retain-II-2024.md
```

Edit: Will resubmit evidence on-chain at a later point as indicated in https://github.com/polkadot-fellows/Evidences/pull/3

### CODE REVIEW

File: evidence/skunert/0001-skunert-retain-II-2024.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+- Date of last report: N/A
+- Area(s) of Expertise/Interest: `Cumulus Node`, `Substrate Node`
+
+## Reporting period
+
+- Start date: 2024/01/08
+- End date: 2024/05/13
+
+## Evidence
+
```
Comment (bkchr) [inline evidence/skunert/0001-skunert-retain-II-2024.md:23]: https://github.com/polkadot-fellows/RFCs/pull/43 your rfc is missing :see_no_evil:

File: evidence/skunert/0001-skunert-retain-II-2024.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+### Projects
+
+Aside smaller contributions, I worked on the following projects:
+
+- **PoV-Reclaim:**
+  - **Description:** Allow parachains to make full use of their PoV-size by reclaiming storage weight that was overestimated due to worst-case benchmarking.
+  - **PRs:** [Runtime Side](https://github.com/paritytech/polkadot-sdk/pull/3002), [Node Side](https://github.com/paritytech/polkadot-sdk/pull/1462), [Integration Guide](https://github.com/paritytech/polkadot-sdk/pull/4244)
+- **Pinning Improvements:**
+  - **Description:** Revisited substrate internal management of notification-based block pinning. Resolves issue where blocks were pinned too long.
```
Comment (bkchr) [inline evidence/skunert/0001-skunert-retain-II-2024.md:32]:
```suggestion
  - **Description:** Revisited Substrate internal management of notification-based block pinning. Resolves issue where blocks were pinned too long.
```

**Review by bkchr — COMMENTED** [review]
Would be nice to also mention some reviews of RFCs or prs in the fellowship runtime repo.

File: evidence/skunert/0001-skunert-retain-II-2024.md (hunk 1 above)
Comment (skunert) [reply evidence/skunert/0001-skunert-retain-II-2024.md:23]: I think the RFC was before the current time period :thinking: . At least opening time, I was not sure what counts.

**Comment by skunert** [comment]
> Would be nice to also mention some reviews of RFCs or prs in the fellowship runtime repo.

I have to admit that is an area for improvement. My expertise is mostly on the node side so I was not as active as I should have been. However I know it's not a very good excuse. I will make sure there is something to show in the next evidence.

File: evidence/skunert/0001-skunert-retain-II-2024.md (hunk 1 above)
Comment (bkchr) [reply evidence/skunert/0001-skunert-retain-II-2024.md:23]: I would include it. I mean you worked on it in the reported time.

**Review by bkchr — APPROVED** [review]

**Comment by skunert** [comment]
Since some days have passed without activity, I updated my on-chain evidence: https://collectives.statescan.io/#/extrinsics/3755964-2

**Pull request merged**